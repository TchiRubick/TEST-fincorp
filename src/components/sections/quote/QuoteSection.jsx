import React from 'react';

import './QuoteSection.scss';

const QuoteSection = () => (
  <div className="section-quote">
    <div className="section-quote__text">
      Title
    </div>
  </div>
);

export default QuoteSection;
