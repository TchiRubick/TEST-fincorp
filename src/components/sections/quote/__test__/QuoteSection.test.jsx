import React from 'react';
import ReactDom from 'react-dom';
import { cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import renderer from 'react-test-renderer';

import QuoteSection from '../QuoteSection';

afterEach(cleanup);

it('renders good', () => {
  const div = document.createElement('div');

  ReactDom.render(<QuoteSection />, div);
});

it('as expected snapshot 1', () => {
  const tree = renderer.create(<QuoteSection />).toJSON();
  expect(tree).toMatchSnapshot();
});
