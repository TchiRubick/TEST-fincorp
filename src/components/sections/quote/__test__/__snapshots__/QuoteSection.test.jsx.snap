// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`as expected snapshot 1 1`] = `
<div
  className="comp-container"
>
  <div
    className="section-quote"
  >
    <div
      className="section-quote__text"
    >
      <img
        alt="quote"
        src="quote.png"
      />
      <span
        className="section-quote__text-content"
      >
        I need to understand what’s going on – it’s my health and I want to feel secure in it. With my previous health insurance, I didn’t know how any of it worked.
      </span>
      <span
        className="section-quote__text-author"
      >
        Matthew Young
      </span>
      <span
        className="section-quote__text-author-location"
      >
        Legal Consultant, United States
      </span>
    </div>
  </div>
</div>
`;
